src/JWT/Builder/JwtBuilder.cs

```diff
@@ -155,20 +155,7 @@ public JwtBuilder WithAlgorithm(IJwtAlgorithm algorithm)
         /// Required to create new token that uses an asymmetric algorithm such as <seealso cref="RS256Algorithm" />.
         /// </remarks>
         /// <returns>Current builder instance</returns>
-        public JwtBuilder WithSecret(string secret)
-        {
-            _secrets = new[] { secret };
-            return this;
-        }
-
-        /// <summary>
-        /// Sets certificates secrets.
-        /// </summary>
-        /// <remarks>
-        /// Required to create new token that uses an asymmetric algorithm such as <seealso cref="RS256Algorithm" />.
-        /// </remarks>
-        /// <returns>Current builder instance</returns>
-        public JwtBuilder WithSecrets(string[] secrets)
+        public JwtBuilder WithSecret(params string[] secrets)
```

Comment:
@CodingMate what would you say?

Reply:
Way better.
How did I miss that, duh :smiley: